src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -71,6 +71,78 @@
 
+   :whiteboard/select            {:binding ["1" "s"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "select")}
+   
+   :whiteboard/pan               {:binding ["2" "p"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "move")}
+   
+   :whiteboard/portal            {:binding "3"
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "logseq-portal")}
+
+   :whiteboard/pencil            {:binding ["4" "d"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "pencil")}
+
+   :whiteboard/highlighter       {:binding ["5" "h"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "highlighter")}
+   
+   :whiteboard/eraser            {:binding ["6" "e"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "erase")}
+   
+   :whiteboard/connector         {:binding ["7" "c"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "line")}
+   
+   :whiteboard/text              {:binding ["8" "t"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "text")}
+
+   :whiteboard/rectangle         {:binding ["9" "r"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "box")}
+
+   :whiteboard/ellipse           {:binding "o"
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "ellipse")}
+
+   :whiteboard/reset-zoom        {:binding "shift+0"
+                                  :fn      #(.resetZoom (.-api ^js (state/active-tldraw-app)))}
+
+   :whiteboard/zoom-to-fit       {:binding "shift+1"
+                                  :fn      #(.zoomToFit (.-api ^js (state/active-tldraw-app)))}
+
+   :whiteboard/zoom-to-selection {:binding "shift+2"
+                                  :fn      #(.zoomToSelection (.-api ^js (state/active-tldraw-app)))}
+
+   :whiteboard/zoom-out          {:binding "shift+dash"
+                                  :fn      #(.zoomOut (.-api ^js (state/active-tldraw-app)) false)}
+
+   :whiteboard/zoom-in           {:binding "shift+="
+                                  :fn      #(.zoomIn (.-api ^js (state/active-tldraw-app)) false)}
+
+   :whiteboard/send-backward     {:binding "["
+                                  :fn      #(.sendBackward ^js (state/active-tldraw-app))}
+
+   :whiteboard/send-to-back      {:binding "shift+["
+                                  :fn      #(.sendToBack ^js (state/active-tldraw-app))}
+
+   :whiteboard/bring-forward     {:binding "]"
+                                  :fn      #(.bringForward ^js (state/active-tldraw-app))}
+
+   :whiteboard/bring-to-front    {:binding "shift+]"
+                                  :fn      #(.bringToFront ^js (state/active-tldraw-app))}
+
+   :whiteboard/lock              {:binding "mod+l"
```

Comment:
Would it be helpful to have a common keybinding prefix for whiteboards shortcuts e.g. `^-W ...`? It could be helpful to avoid conflicts but more importantly would provide consistent, memorable whiteboard shortcuts. It's unlikely that future whiteboard shortcuts will consistently get mod+LETTER where LETTER is the first letter of the feature name

Reply:
I though about this a lot while working on the enhancement. I think the following reasons outweigh the benefits of this, but I am open to discuss this further
- The starting letter of our existing chords is always the action (go, toggle, close). This is a different type of convention, that would make the shortcuts hard to predict and inconsistent. For example, using "t g" for :whiteboard/toggle-grid might make more sense
- It feels a little redundant, since the context is the whiteboard
- Most of those shortcuts are the same across similar applications
- Some of the editor shortcuts also work on whiteboards
- They are already introduced to our users

Reply:
> The starting letter of our existing chords is always the action (go, toggle, close). This is a different type of convention, that would make the shortcuts hard to predict and inconsistent.

Agree that current chords only use verbs except for `(n)ew` but introducing a different type of grouping would be helpful. Verbs aren't going to cover most groupings as evidenced by all the shortcuts that don't use those chords.  Another version of my suggestion would be to use "w" for whiteboard shortcuts.

Overall, I'm trying to encourage more namespacing/chording/grouping of shortcuts since we have no clear criteria for top-level keybindings. Without a clear criteria, these top-level keybindings are probably going to feel arbitrary to users. `Cmd-k` and `Cmd-f` are global actions that are useful to all users. `Cmd-l` and soon to be `Cmd-u` from https://github.com/logseq/logseq/pull/9245/files#diff-b3ebcab17041ddd88e90b63f3f6edc1db33b5dd469a91fed5ed3d4432bfe677a only work in certain contexts. If local actions have top-level keybindings then users will always wonder why their preferred local action isn't a top-level keybinding e.g. cmd+l for something related to linked references or local http server